docs/DocumentList.md

```diff
@@ -0,0 +1,29 @@
+<!--- SPDX-License-Identifier: Apache-2.0 -->
```

Comment:
Having a file with references to each of the document files is good. Were you aware of the top level CONTRIBUTING.md file? Their functionality seems somewhat redundant, and the info in both could be combined.

Reply:
Yes, they are redundant. I will open an issue to record all the issues for document.